client/src/store/graphql/files/queries.tsx

```diff
@@ -0,0 +1,64 @@
+import { gql } from '@apollo/client';
+
+export const FILES_QUERY = gql`
+{
+  Patient(id:"QA-PA-00128"){
```

Comment:
I'm confused here. is this for a test? Why there is a hardcoded patient id in the query? is this query used at all?